pom.xml

```diff
@@ -223,274 +111,85 @@
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-parser</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-script</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-shared-resources</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-svg-dom</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-svggen</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-transcoder</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-util</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.apache.xmlgraphics</groupId>
-                    <artifactId>batik-xml</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>com.smartbear</groupId>
-                    <artifactId>common-ui</artifactId>
-                </exclusion>
-
-                <exclusion>
-                    <groupId>com.optimizely.ab</groupId>
-                    <artifactId>core-api</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>com.optimizely.ab</groupId>
-                    <artifactId>core-httpclient-impl</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>xmlpull</groupId>
-                    <artifactId>xmlpull</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>xpp3</groupId>
-                    <artifactId>xpp3_min</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>jakarta.xml.bind</groupId>
-                    <artifactId>jakarta.xml.bind-api</artifactId>
-                </exclusion>
-
-                <exclusion>
-                    <groupId>com.google.guava</groupId>
-                    <artifactId>guava</artifactId>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
                 </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.smartbear</groupId>
+            <artifactId>ready-api-soapui-ng</artifactId>
```

Comment:
This is a duplicate of the previous dependency.

Reply:
which line?
i see "ready-api-soapui-pro" as previous.
these dependencies needed to compile project correctly. Later of course these depencies willl be inside readyapi.
i've added "ready-api-soapui-pro" but exclude all dependencies for "ready-api-soapui-pro". The same for "ready-api-soapui-ng"

Reply:
![image](https://user-images.githubusercontent.com/11975824/152543477-e3fae6ad-bc61-48cb-8607-70970d0fb361.png)

Reply:
👍